src/redist/redist.csproj

```diff
@@ -148,6 +148,7 @@
   </Target>
 
   <Target Name="PublishLzmaArchive"
+          Condition="'$(CLIBUILD_SKIP_LZMA)' != 'true'"
```

Comment:
@ellismg I thought we already had a knob to turn this off that we used for build from source.